rand_gen/src/main.rs

```diff
@@ -0,0 +1,31 @@
+extern crate rand;
+use rand::Rng;
+use std::fs;
+
+
+fn main() {
+    
+    let mut count=0;
+    
+    let mut paths: Vec<_> = fs::read_dir("./").unwrap()
+    .map(|r| r.unwrap())
+    .collect();
+    paths.sort_by_key(|dir| dir.path());
```

Comment:
Sorting the paths is unnecessary

Reply:
I moved everything to arclightning_backend/src/splash.rs. I don't know how to delete rand_gen.